client/landing/stepper/declarative-flow/internals/steps-repository/goals/goals.tsx

```diff
@@ -77,47 +75,5 @@ export const useGoals = ( isAddedGoalsExp: boolean ): Goal[] => {
 		return shuffleArray( goals );
 	}, [ translate ] );
 
-	const goals = [
-		{
-			key: SiteGoal.Write,
-			title: translate( 'Write & Publish' ),
-		},
-		{
-			key: SiteGoal.Sell,
-			title: translate( 'Sell online' ),
-		},
-		{
-			key: SiteGoal.Promote,
-			title: translate( 'Promote myself or business' ),
-		},
-		{
-			key: SiteGoal.DIFM,
-			title: translate( 'Let us build your site in 4 days' ),
-			isPremium: true,
-		},
-		{
-			key: SiteGoal.Import,
-			title: translate( 'Import existing content or website' ),
-		},
-		{
-			key: SiteGoal.Other,
-			title: translate( 'Other' ),
-		},
-	];
-
-	/**
-	 * Hides the DIFM goal for RTL locales.
-	 */
-	const hideDIFMGoalForUnsupportedLocales = ( { key }: Goal ) => {
-		if ( key === SiteGoal.DIFM && isLocaleRtl( locale ) ) {
-			return false;
-		}
-		return true;
-	};
-
-	if ( isAddedGoalsExp ) {
-		return addedGoalsExpResult;
-	}
-
-	return goals.filter( hideDIFMGoalForUnsupportedLocales );
+	return addedGoalsExpResult;
```

Comment:
@p-jackson there is no experiment anymore around these goals so should we rename this variable?